### DISCUSSION TITLE
Results per page css misaligned

### DISCUSSION THREAD

**Discussion by mallanza** [rappasoft/laravel-livewire-tables#1633] [Q&A]
Wondering if anyone encountered this issue? Results per page css misaligned

![image](https://github.com/rappasoft/laravel-livewire-tables/assets/153227702/1d3fe0fa-d121-46ea-8c63-da08c3f78a53)

To counter, I had to do custom css 

```
.tableWrapper div div div div div div div{
    margin-bottom: 0 !important;
}
```

**Comment by lrljoe** [comment]
I haven't come across this, can you please confirm:

Which version of the package you're using (v2.x or v3.x)
Which theme you're using, Tailwind 2, Tailwind 3, Bootstrap 4, or Bootstrap 5?
Whether you're using Vite/Mix as a bundler?

**Comment by mallanza** [comment]
I use 
- v3.2
- Tailwind 3
- Vite

Also, I see the same issue with results per page misaligned in this discussion - https://github.com/rappasoft/laravel-livewire-tables/discussions/1183
